reconcile/utils/ocm.py

```diff
@@ -996,6 +997,32 @@ def delete_external_configuration_label(self, cluster, label):
         )
         self._delete(api)
 
+    def _get_subscription_labels_api(self, cluster: str) -> str:
+        cluster_id = self.cluster_ids[cluster]
+        api = f"{CS_API_BASE}/v1/clusters/{cluster_id}"
+        subscription_api = self._get_json(api)["subscription"]["href"]
+        return f"{subscription_api}/labels"
+
+    def is_cluster_admin_enabled(self, cluster: str) -> bool:
+        api = self._get_subscription_labels_api(cluster)
+        subcription_labels = self._get_json(api).get("items")
+
+        for sl in subcription_labels:
```

Comment:
Perhaps `subcription_labels or []` to make it more robust?

Reply:
`items` returns an empty list, but sure